setup/azurecmdfiles.wxi

```diff
@@ -418,15 +418,6 @@
                         <Component Id="cmp4B5A611FF761BA928A53E0E56FB58D78" Guid="*">
```

Comment:
I have build failures on my clean devbox because it could not find the dependencies that I removed here. After removing them, the build got fixed. I will revert this, but it looks like these dependencies are not needed and should be removed.